packages/server/src/config.ts

```diff
@@ -111,6 +111,31 @@ function baseUrlFromEnv(env: EnvVariables, appPort: number): string {
 	}
 }
 
+function ldapConfigFromEnv(env: EnvVariables): LdapConfig[] {
+	return [
+		{
+			enabled: env.LDAP_1_ENABLED,
+			userCreation: env.LDAP_1_CREATE_USER,
```

Comment:
Same here - we try to use the same names consistently for config and env variables, it makes it easier to know what is what. Can I suggest `USER_AUTO_CREATION` to make it clear it means users are going to be automatically created when they don't already exist?